components/brave_shields/browser/https_everywhere_recently_used_cache.h

```diff
@@ -3,83 +3,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_COMPONENTS_BRAVE_SHIELDS_BROWSER_HTTPS_EVERYWHERE_RECENTLY_USED_CACHE_H_
-#define BRAVE_COMPONENTS_BRAVE_SHIELDS_BROWSER_HTTPS_EVERYWHERE_RECENTLY_USED_CACHE_H_
+#ifndef BRAVE_COMPONENTS_BRAVE_SHIELDS_BROWSER_HTTPS_EVERYWHERE_RECENTLY_USED_CACHE_H_  // NOLINT
+#define BRAVE_COMPONENTS_BRAVE_SHIELDS_BROWSER_HTTPS_EVERYWHERE_RECENTLY_USED_CACHE_H_  // NOLINT
```

Comment:
I'm surprised header guards aren't excluded from line length lint.

Reply:
I had the same situation. Guards are excluded, but they need to be correct. So I needed to first use correct naming for the guard(suggested by linter) and then line length error went away

Reply:
ok I think you can remove the NOLINT in that case

Reply:
Removed.